Source/Diagnostics/ReducedDiags/ParticleHistogram2D.cpp

```diff
@@ -0,0 +1,302 @@
+//
+// Created by juliette on 13/04/23.
+//
+
```

Comment:
```suggestion
```suggestion
/* Copyright 2023 The WarpX Community
 *
 * This file is part of WarpX.
 *
 * Authors: Juliette Pech, Axel Huebl
 * License: BSD-3-Clause-LBNL
 */
```
```